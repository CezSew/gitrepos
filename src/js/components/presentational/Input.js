import React, { Component } from 'react';

class Input extends Component {
    constructor(props) {
        super(props);
    }
    render() {
        return (
            <input id="user-input" className="form-control" type="text" name="user" placeholder="User" autoComplete="off"/> 
        );
    }
}


export default Input;