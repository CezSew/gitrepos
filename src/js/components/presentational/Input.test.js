import React from 'react';
import { shallow } from 'enzyme';
import Input from './Input';


describe('rendering', () => {
    let wrapper
    beforeEach(() => {
        wrapper = shallow(<Input />)
    })
    it('should render an input', () => {
        expect(wrapper.find('#user-input')).toHaveLength(1)
    })
})