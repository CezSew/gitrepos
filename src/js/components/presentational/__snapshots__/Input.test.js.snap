// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Rendering simple component 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <input
    autoComplete="off"
    className="form-control"
    id="user-input"
    name="user"
    placeholder="User"
    type="text"
  />,
  "nodes": Array [
    <input
      autoComplete="off"
      className="form-control"
      id="user-input"
      name="user"
      placeholder="User"
      type="text"
    />,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_context": Object {},
    "_element": <Input />,
    "_forcedUpdate": false,
    "_instance": Input {
      "context": Object {},
      "props": Object {},
      "refs": Object {},
      "state": null,
      "updater": Updater {
        "_callbacks": Array [],
        "_renderer": [Circular],
      },
    },
    "_newState": null,
    "_rendered": <input
      autoComplete="off"
      className="form-control"
      id="user-input"
      name="user"
      placeholder="User"
      type="text"
    />,
    "_rendering": false,
    "_updater": Updater {
      "_callbacks": Array [],
      "_renderer": [Circular],
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Input />,
}
`;
